comit/src/network/protocols/orderbook.rs

```diff
@@ -273,10 +272,21 @@ pub type OrderId = Uuid;
 #[derive(Debug, Clone, PartialEq)]
 pub struct MakerId(PeerId);
 
-impl MakerId {
-    /// Returns a clone of the inner peer id.
-    pub fn peer_id(&self) -> PeerId {
-        self.0.clone()
+impl From<PeerId> for MakerId {
+    fn from(id: PeerId) -> Self {
+        MakerId(id)
+    }
+}
+
+impl From<MakerId> for PeerId {
+    fn from(id: MakerId) -> Self {
+        id.0
+    }
+}
+
+impl Display for MakerId {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.0.to_string())
```

Comment:
```suggestion
        write!(f, "{}", self.0)
```
This should be enough :)